deployments/systemd/nvidia-cdi-refresh.service

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[Unit]
+Description=Refresh NVIDIA CDI specification file
+
+[Service]
+Type=oneshot
+# The 30-second delay ensures that dependent services or resources are fully initialized.
+ExecStartPre=/bin/sleep 30
+ExecStart=/usr/bin/nvidia-ctk cdi generate --output=/var/run/cdi/nvidia.yaml
```

Comment:
Question: Should this path depend on the installation locations?

Reply:
Good point, yes. umm let me think how to mod this

Reply:
umm after checking we `/usr/bin` set as default for both deb/rpm, and we don't provide macros to change the install path. So having this hardcoded here looks ok. WDYT?

Reply:
Then install command installs to:
```
install -m 755 -t %{buildroot}%{_bindir} nvidia-ctk
```
If this is always `/usr/bin` then we don't neet to update this.

Reply:
According to the documentation, we can leave it as is. - https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/#macros_installation